images/nginx/rootfs/build.sh

```diff
@@ -95,6 +93,7 @@ clean-install \
   dumb-init \
   gdb \
   valgrind \
+  luajit \
```

Comment:
why is this? we build luajit from source

Reply:
Because of this https://packages.debian.org/buster/luajit. I rollbacked this change